### DISCUSSION TITLE
Fit API Depreciated

### DISCUSSION THREAD

**Discussion by WillScott73** [YorkshireIoT/ha-google-fit#197] [General]
Hi

I note that the Google Fit API is depreciated and will be gone by the end of the year.

https://developers.google.com/fit/android

How does this impact this awesome Home Assistant integration?

**Comment by YorkshireIoT** [comment]
As far as I can tell, it looks like they're just deprecating the Android specific API.

The REST API looks like it's not going anywhere (yet):

> Q: What will happen to the Google Fit REST API and Fitbit Web API?
> We will continue to invest in and grow our Web APIs. These APIs are an essential part of our mission to help developers build better health, fitness, and wellness technologies for their users.

Source: https://developer.android.com/health-and-fitness/guides/health-connect/migrate/comparison-guide#google-fit

**Reply by KenDodman** [reply]
> Caution: The Fit REST API will only be supported through June 30, 2025. We recommend integrating with Health Connect or the Fitbit Web API.
